### ISSUE TITLE
Installer doesn't work with nvme disks

### ISSUE THREAD

**Issue by jnread** [deadhead420/arch-linux-anywhere#121]
NVMe drive partitions are, eg:

```
nvme0n1p1
nvme0n1p2
etc
```

The installer misses the 'p' after partitioning, ie tries to use

```
nvme0n11
nvme0n12
etc
```

This results in an 'error during partitioning' failure.

symlinking 

> nvme0n11 --> nvme0n1p1

 etc allows partitioning to complete

**Comment by deadhead420** [comment]
This should be fixed in the latest version I'm about to release but I have no way to test.

**Comment by MitchRazga** [comment]
Was this fix applied to 2.2.5? I've just attempted an auto partition and the installer is still dropping the 'p' in nvme0n1p1, nvme0n1p2, nvme0n1p3 etc.

**Comment by pnedkov** [comment]
#140

**Comment by deadhead420** [comment]
Should be fixed now for auto partitioning thanks to pnedkov. Not sure if this issue is present for manual partitioning. Looking into it now.

**Comment by mrobinsn** [comment]
I can confirm that #140 fixed the issue for me. Just installed successfully on a new XPS 15 (9560) with NVMe drive.

**Comment by Rukenshia** [comment]
@deadhead420 Installation does not work for me, I have an intel 600p series nvme ssd. I am using the latest release (2.2.5) of arch anywhere, my CPU is a i7 7700k. I just get the message `An error was detected during partitioning`. Looking at the partition table, it does seem to be created fine. I tried setting it up with automatic partitioning with encrypted lvm and without it. neither worked for me. In manual mode, I can't even change the mount points properly (can't set the / mountpoint manually, getting continuous errors there).

**Comment by pnedkov** [comment]
NVME was fixed after 2.2.5 was released.

**Comment by Rukenshia** [comment]
Oh, thanks @pnedkov. I would really appreciate getting a new build as soon as possible then. Don't really want to go through the pain of installing it manually.

**Comment by pnedkov** [comment]
@Rukenshia , currently there are issues with some languages other than English. If you are using English you should not face any problems. Give it a try and I will appreciate it if you give us some feedback how it went.

**Comment by Rukenshia** [comment]
Alright, I'll try it tomorrow and report back.

**Comment by pnedkov** [comment]
@Rukenshia , a quick update: Wait for #162 and #163 to get merged before start installing.

**Comment by Rukenshia** [comment]
Just tried it with the latest master @pnedkov and it does not work. The dialog also seems to be broken right now as I can see the color sequences on my screen. On a dialog that asks me to confirm that I want to erase the drive, it says it has a size of 0GiB.

**Comment by pnedkov** [comment]
@Rukenshia , could you please try again? I think #162 was not merged into deadhead's master at the time you tried. I got positive feedback from one of the maintainers that after merging #162 and #163 the installer is fully operational.

**Comment by Rukenshia** [comment]
@pnedkov, I just tried it again and it did not work.

I am attaching a few images of what I can see on my monitor. As a side note, it seems as if the commands displayed during the "partitioning process" exit way too fast, if I run mkfs.ext4 manually the command takes way longer.

![image](https://cloud.githubusercontent.com/assets/1899599/22944967/d7bf2ea8-f2f2-11e6-9d47-ca60305d3023.png)
![image](https://cloud.githubusercontent.com/assets/1899599/22944974/dbadc538-f2f2-11e6-8211-9f6aaaa1219c.png)
![image](https://cloud.githubusercontent.com/assets/1899599/22944982/e081f408-f2f2-11e6-8620-40fa1fccb3c2.png)

**Comment by Daylien** [comment]
Just one question does emmc disks equal to nvme disks ? I have an similar error on my notebook with an emmc disk. The disk failed to mount after the partitioning. Tried it on two notebooks with emmc's disks.

**Comment by pnedkov** [comment]
@Rukenshia , @Daylien , you are right. Non sdX devices are broken. Will fix as soon as I can.

**Comment by pnedkov** [comment]
#166 

Please give it a try once it is merged.

**Comment by deadhead420** [comment]
Let me know if the fix works and I'll close this issue

**Comment by Rukenshia** [comment]
I'll try it tomorrow and report back.

**Comment by deadhead420** [comment]
@Rukenshia sounds good. Thanks!

**Comment by Rukenshia** [comment]
Works now @deadhead420 @pnedkov. Thanks!

**Comment by deadhead420** [comment]
@Rukenshia Glad to hear!! Closing issue. Thanks @pnedkov for your work on this!!

**Comment by pnedkov** [comment]
#170 

Just found another problem and fixed it. My testing shows it is working but please give it a spin.
